src/jit.rs

```diff
@@ -980,6 +986,22 @@ impl JitCompiler {
         })
     }
 
+    #[cfg(test)]
+    #[allow(dead_code)]
+    pub(crate) fn new_mock() -> Self {
```

Comment:
Should be moved over in x86.rs `mod test { ... }` as this is only used by the instruction emitter.